plugin-server/src/main/ingestion-queues/session-recording/session-recordings-consumer.ts

```diff
@@ -312,17 +312,6 @@ export class SessionRecordingIngester {
             }
         }
 
-        // added for incident 367 https://posthog.slack.com/archives/C08LPFEAUNR
```

Comment:
@benjackwhite yep, we moved the checking into rust capture, so new messages with this nonsense in session id wouldn't get this far